### ISSUE TITLE
cannot overwrite year

### ISSUE THREAD

**Issue by fdlk** [LiquidFun/aoc_tiles#3]
I have a single year in my repo and in the pre commit config I have tried to override the year:
```
repos:
  - repo: https://github.com/LiquidFun/aoc_tiles
    rev: 0.5.4
    hooks:
      - id: aoc-tiles
        args:
          - --overwrite-year=2023
```
However, I get an error when I commit:
```
aoctiles: error: argument --overwrite-year: invalid Optional value: '2023'
```
I've tried without the = as well, so `- --overwrite-year 2023` but then I get leaderboardNone.html with a 404 in it

**Comment by mSpoel** [comment]
I run into the same issue, something is off with setting the optional int

**Comment by LiquidFun** [comment]
Can you test it in `0.5.6`? It should be fixed now

**Comment by LiquidFun** [comment]
Fixed in 0.5.6
